Expecto/Expect.fs

```diff
@@ -228,7 +229,77 @@ let containsAll (actual : _ seq)
         %s
         Extra elements in `actual`:
         %s"
-              format (formatSet axs) (formatSet exs) (formatSet missing) (formatSet extra)
+              format (formatResult axs) (formatResult exs) (formatResult missing) (formatResult extra)
+  |> Tests.failtest
+
+let inline private except(baseMap: Map<_,int>, exceptMap: Map<_,int>, isExcept: bool) =
```

Comment:
The code style should be:

```
let fn (para : type) (para2 : type2) =
```

with:

 - spaces after parameter names
 - spaces after function name
 - preferring curried application instead of tuples

Could we also rename these parameters? What does "exceptMap" mean? Can we make it more semantically appropriate?